### PR TITLE
WIP feat: intial header setup

### PR DESCRIPTION
**Pull request by laurafitzgerald**
## Motivation
Not sure which JIRA to put this under yet.
This may cover some work from this one https://issues.jboss.org/browse/AEROGEAR-8375

## What
Add React Routing and Header

## Why
See Motivation

## Checklist:

- [ ] Code has been tested locally by PR requester
- [ ] Changes have been successfully verified by another team member 

## Progress

- [x] Add React Router
- [ ] Fill out Header

## Additional Notes

This is a basic WIP start as was mostly lifted from mobile-developer-console. Pushing for review from @damienomurchu

### CODE REVIEW

File: ui/src/App.js [hunk 1]
```diff
@@ -1,15 +1,25 @@
...
-      <div className="App">
-        <LandingPage />
-      </div>
-    );
+    <Router>
+    <div className="App">
+      <Header />
+      <Switch>
+        <Route exact path="/overview" component={LandingPage} />
+        <Route exact path="/mobileclient/:id" component={AppView} />
```
Comment (laurafitzgerald) [inline ui/src/App.js]:
``` suggestion
        <Route exact path="/app/:id" component={AppView} />
```

**Comment by laurafitzgerald** [comment]
@damienomurchu pushed some new changes here. I'm not sure if we need the container files. I've moved the Nav Bar Code into one file without the container file as an example.

**Comment by laurafitzgerald** [comment]
@damienomurchu I've updated this to follow the container structure. I also updated the Routing as suggested. Are you free to review this now?

**Comment by damienomurchu** [comment]
@laurafitzgerald Header looks good, just ran it locally. You might have a messy rebase to try land this on my PR branch here, so will ping you offline about how we can best land this work into the codebase. 

I will take a look at the code shortly, and then will look my landing-page-ui & see if I can get that PR in a state where it's mergeable. 

Good work :+1: :)

**Comment by laurafitzgerald** [comment]
@damienomurchu agreed on the messy rebase. I think the best approach here is to branch from master. I've had a look at how you are doing the folder structure which lines up with where I was going with it.

**Comment by damienomurchu** [comment]
@laurafitzgerald that's good re folder structure. As per Wei's suggestion, I'm moving all components that interact with the redux store into `src/containers/`. That process is still ongoing, and welcome your thoughts once they're all moved

**Pull request closed without merging**